Wireframe/css/style.css

```diff
@@ -0,0 +1,68 @@
+*{
+    margin:0;
+    padding:0;
+
+}
+body{
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    line-height: 2.5rem;  
+    font-size: 1rem;
+    text-align: center;
+    align-items: center;
+    margin: 2.5rem 5rem;
+    border: 2rem solid white;
+    border-radius: 0.2rem;
+   
+}
+
+.header-container{
+    background-color: white;
+    border-bottom:0.5rem solid rgb(207, 205, 202);
+    border-radius: 0.3rem;
+}
+
+.main-section{ 
+    padding: 2rem 0;
+    border-radius: 0.3rem;
+}
+.left-section{
+    width: 45%;
+    float: left;
+}
+.right-section{
+    float: right;
+    width: 45%;
+}
+h1,h2,h3,h4{
+    font-weight: bold;
+    color:rgb(98, 98, 97)
+}
+a:link{
+    text-decoration: none;
+    cursor: pointer;
+    color: rgb(57, 57, 138);
+   
+}
+  a:visited {
+    color: rgb(52, 72, 158);
+  }
+  a:hover {
+    color: hotpink;
+  }
+  a:active {
+    color: blue;
+  }
+
+.footer-container{
+    margin-top:16rem;
```

Comment:
This margin top feels dangerous - what is your goal with this rule?

Reply:
I use it to align the footer at the specific bottom. The footer element becomes closer to the last element of the man content  to have more space between I use margin. That it's why I use. please could you recommend me what can I use instead ?

Thank you!